src/components/navBar/navlink.js

```diff
@@ -0,0 +1,38 @@
+import styles from '@/components/navBar/navBar.module.scss';
+
+const links = [
+    {
+      id:1,
+      name:'Home',
+      link:'https://www.realdevsquad.com/',
+      tabStyle:styles.homeTab
+    },
+    {
+      id:2,
+      name:'Welcome',
+      link:'https://welcome.realdevsquad.com/'
+    },
+    {
+      id:3,
+      name:'Events',
+      link:'https://www.realdevsquad.com/events.html'
+    },
+    {
+      id:4,
+      name:'Members',
+      link:'https://members.realdevsquad.com/'
+    },
+    {
+      id:5,
+      name:'Crypto',
+      link:'https://crypto.realdevsquad.com/'
+    },
+    {
+      id:6,
+      name:'Status',
+      link:'https://status.realdevsquad.com/',
+      linkStyle:styles.activeTab
```

Comment:
Is there any specific reason for making this property name different from that on line 8 ??

Reply:
Got the reason why it is so